src/components/HomeSection.jsx

```diff
@@ -61,6 +61,7 @@ HomeSection.defaultProps = {
   alignItems: 'flex-start',
   direction: 'row',
   tab: '',
+  buttons: '[]',
```

Comment:
Look at line 73, it spells out `buttons`' `PropTypes` is an `arrayOf` stuff. So this should be an array, not a string.
```suggestion
  buttons: [],
```